docs/tutorials/varlink_remote_client.md

```diff
@@ -1,89 +0,0 @@
-# Podman varlink remote-client tutorial [DEPRECATED]
-
-## What is the varlink client
-
```

Comment:
I'd keep this, but add something like:
```
### **NOTE:** as of Podman v3.0 the varlink client has been removed and is only usable in versions prior to Podman v3.0.
```
People may still use v2.0 for a while (RHEL 8.2 and earlier) and will want to get to this.